### PR TITLE
Preparing forum launch

### PR DESCRIPTION
**Pull request by simonsan**
I've added links to linux, windows packages and the main repository on the left side.
Linked the forum and edited some links on the right side.

I've also updated the text on the right side to the one of the new website.

Updated main.css correspondingly to get three buttons to the middle of the left side of the page.

Updated links to `https://<subdomain>.openage.dev` for easier change of Download links, Redirects, etc. in the nearer future.

Put the main contact methods more in the front (right side) and moved the other links down.

**The text describing the project is a draft and needs to be discussed in the group.**

Screenshot latest version:
2020-03-03 05_10_49--Screenshot

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -17,28 +17,31 @@
         <div id="main">
             <div id="left">
                 <img src="pics/openageinverse.svg" alt="openage" id="logo">
-                <h1>A free (as in <i>freedom</i>) clone of the Age of Empires II engine</h1>
+                <h1>Free cross-platform RTS game engine based on Age of Empires mechanics</h1>
```
Comment (heinezen) [inline index.html]: I would rather like the phrase "[...] RTS engine providing Age-of-Empires-style game mechanics" because that makes it a bit clearer what openage offers.

File: index.html [hunk 2]
```diff
@@ -17,28 +17,31 @@
...
             <div id="rightcontainer"><div id="right">
-                <p>You know Age of Empires. You know its bugs. You know the fun playing it.</p>
-                <p>We are creating a free (as in freedom) clone of the Age of Empires II engine in C++17 from scratch (GPLv3+), using "modern" technologies like OpenGL, Vulkan, Python, Qt5, CMake, <a href="https://github.com/SFTtech/nyan/">nyan</a>, ...</p>
-                <p>This allows people using GNU/Linux, BSD, macOS or Windows to play the game natively, minus features, plus additional epic bugs (or was it vice versa?).</p>
-                <p>Until we have free replacement assets, we can import original media files provided by the user.</p>
-                <p>openage is a huge and ambitious project, run by volunteers: Students and professionals from all over the world. We are glad about every helping hand, so the only requirement is your enthusiasm :)</p>
-                <p>Join us and help to create an awesome game!</p>
+				<p>We are creating a free (as in freedom, GPLv3+) cross-platform RTS game engine based on Age of Empires mechanics in C++17. We use "modern" technologies like OpenGL/GLSL, Python, Qt5, CMake, ...</p>
```
Comment (heinezen) [inline index.html]: 1. Maybe this starts better with "openage is [...]" instead of "We are creating [...]" since it is not as clear who *we* are.
2. "free (as in freedom)" could link to the GPL3 text instead of mentioning it.
3. nyan could get mentioned and linked in the list of technologies

File: index.html [hunk 3]
```diff
@@ -17,28 +17,31 @@
...
+				<p>This allows people using GNU/Linux, BSD, macOS or Windows to play the game natively.</p>
+				<p>We aim to create a game engine that provides the original look and feel. Until we have free replacement assets, we can convert original media files provided by the user to an format compatible to openage.</p>
```
Comment (heinezen) [inline index.html]: Would rephrase this because it is based on the previous mention of the "Age of Empires clone" whereas the scope is now much bigger. I would suggest merging these sentences into something like this:

> Our aim is to make openage a platform for the original Age of Empires games providing the same look and feel, but with more features for modding and multiplayer.

File: index.html [hunk 4]
```diff
@@ -17,28 +17,31 @@
...
+
+				<p><b>A look into the future:</b> openage has a lot of improvements planned. It is going to add completely new possibilities and will provide better performance due to different design decisions being made from the beginning on, e.g. being multithreaded and using modern technologies like Vulkan rendering. Our self-designed entity-component-system <a href="https://github.com/SFTtech/nyan/">nyan</a> will help us exposing a Modding API making openage more easy for modders to use features of openage in their mods.</p>
```
Comment (heinezen) [inline index.html]: We should not promise too much here, since the original games and openage are not that comparable. I would rather pick up on the sentence from the previous comment and explain them a tiny bit more. Maybe one sentence for each thing

1. modding: Modding API, multiple mods at the same time
2. multiplayer: dedicated servers, more than 8 players

File: index.html [hunk 5]
```diff
@@ -17,28 +17,31 @@
...
+
+				<p>We always welcome new contributors! openage is a huge and ambitious project. There is more work than our small core dev team mainly run by volunteers can handle. The openage community is very open, welcoming and supportive. If you’ve got the enthusiasm, you’re all welcome.</p>
```
Comment (heinezen) [inline index.html]: I would keep this much shorter and not bother with much details here or scare people with the "huge and ambitious" line. This would be my proposition:

> openage is a community project. We welcome every contribution, the only requirement is your enthusiasm. [Get in touch](https://github.com/SFTtech/openage#contact) with us if you want to help :)

File: index.html [hunk 6]
```diff
@@ -17,28 +17,31 @@
...
+				<p>Get in contact with us if you have any issues getting started.</p>
```
Comment (heinezen) [inline index.html]: see above

**Review by heinezen — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (simonsan) [reply index.html]: Yeah, sounds better. Also "providing" sound much more **actively supportive** than a passive "based" on.

File: index.html (hunk 4 above)
Comment (simonsan) [reply index.html]: Updated the screenshot in the initial post, take a look. For the Modding stuff might be better if you write a short sentence. 👍

Comment (heinezen) [reply index.html]: **Modding**: openage uses an open API powered by our human-readable configuration language [nyan](https://github.com/SFTtech/nyan).

**Multiplayer**: We implement a client-server architecture with dedicated servers that supports more than 8 players. The overarching system will provide matchmaking, lobbies, server discovery and other community features.

**Comment by heinezen** [comment]
The comment I wrote needs to be addressed. Otherwise it's good to go :)

**Review by heinezen — APPROVED** [review]

**Comment by TheJJ** [comment]
Thanks!
The content looks much better now. We can do further improvements later on.

Unfortunately, on my screen it doesn't look very well layouted:
Screenshot_2020-03-13 openage - free, modern RTS game engine

To improve things, I added a commit to the branch, which restores the full responsive design that supports arbitrary screen aspect ratios and the page will always™ look good.
